api/notd/api/gallery_v1.py

```diff
@@ -92,4 +93,9 @@ async def get_gallery_user_owned_collections(registryAddress: str, userAddress:
         ownedCollections = await galleryManager.get_gallery_user_owned_collections(registryAddress=registryAddress, userAddress=userAddress)
         return GetGalleryUserOwnedCollectionsResponse(ownedCollections=(await responseBuilder.gallery_owned_collections_from_models(ownedCollections=ownedCollections)))
 
+    @router.get('/collections/{galleryAddress}/overlaps')
+    async def get_list_collection_overlaps(galleryAddress: str) -> GetCollectionOverlapsResponse:
```

Comment:
for listing endpoints (where the result is a list of things) the "method" of the api should be considered as "List", so update the request/response objects and function name - no get, just List* and list_*